File: Drivo.Entities/InternalExamEntity.cs
namespace Drivo.Entities
{
    public class InternalExamEntity : EventEntity
    {
        public virtual StudentEntity? Student { get; set; }
        public int? StudentId { get; set; }
        public virtual InstructorEntity? Instructor { get; set; }
        public int? InstructorId { get; set; }
    }
}

File: Drivo/src/entities/InternalExamEntity.ts
import { EventEntity } from "./event.entity"
import { InstructorEntity } from "./InstructorEntity";
import { StudentEntity } from "./StudentEntity";

export class InternalExamEntity extends EventEntity
{
    student: StudentEntity | null;
    studentId: number | null;
    instructor: InstructorEntity | null;
    instructorId: number | null;
}